integration/kubernetes/k8s-number-cpus.bats

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+issue="https://github.com/kata-containers/runtime/issues/2186"
+
+setup() {
+	skip "test not working see: ${issue}"
+	export KUBECONFIG="$HOME/.kube/config"
+	pod_name="cpu-test"
+	container_name="c1"
+	get_pod_config_dir
+}
+
+@test "Check number of cpus" {
+	skip "test not working see: ${issue}"
+	# Create pod
+	kubectl create -f "${pod_config_dir}/pod-number-cpu.yaml"
+
+	# Check pod creation
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	# Expect some time to get nproc information
+	time="10"
+	sleep "$time"
+
+	# Get number of cpus
+	max_number_cpus="3"
+	number_cpus=$(kubectl exec -ti pod/"$pod_name" -c "$container_name" nproc | sed 's/[[:space:]]//g')
```

Comment:
given the nature of the problem, maybe put the `exec` in a loop that is significantly larger than `max_number_cpus`? Only testing it the once and having `max_number_cpu`s hard wired here and relying on the layout in the yaml file feels a little 'borderline' to me.